server/blob/blob-api/src/test/java/org/apache/james/blob/api/BlobStoreContract.java

```diff
@@ -175,4 +178,51 @@ default void readShouldReturnBigSavedData() {
 
         assertThat(read).hasSameContentAs(new ByteArrayInputStream(TWELVE_MEGABYTES));
     }
+
+    @Test
+    default void deleteShouldNotThrowWhenBlobDoesNotExist() {
+        assertThatCode(() -> testee().delete(testee().getDefaultBucketName(), blobIdFactory().randomId()).block())
+            .doesNotThrowAnyException();
+    }
+
+    @Test
+    default void deleteShouldDeleteExistingBlobData() {
+        BlobId blobId = testee().save(testee().getDefaultBucketName(), SHORT_BYTEARRAY).block();
+        testee().delete(testee().getDefaultBucketName(), blobId).block();
+
+        assertThatThrownBy(() -> testee().read(testee().getDefaultBucketName(), blobId))
+            .isInstanceOf(ObjectStoreException.class);
+    }
+
+    @Test
+    default void deleteShouldBeIdempotent() {
+        BlobId blobId = testee().save(testee().getDefaultBucketName(), SHORT_BYTEARRAY).block();
+        testee().delete(testee().getDefaultBucketName(), blobId).block();
+
+        assertThatCode(() -> testee().delete(testee().getDefaultBucketName(), blobId).block())
+            .doesNotThrowAnyException();
+    }
+
+    @Test
+    default void deleteShouldNotDeleteOtherBlobs() {
+        BlobId blobIdToDelete = testee().save(testee().getDefaultBucketName(), SHORT_BYTEARRAY).block();
+        BlobId otherBlobId = testee().save(testee().getDefaultBucketName(), ELEVEN_KILOBYTES).block();
+
+        testee().delete(testee().getDefaultBucketName(), blobIdToDelete).block();
+
+        InputStream read = testee().read(testee().getDefaultBucketName(), otherBlobId);
```

Comment:
The testee().getDefaultBucketName() into a local variable could be extracted into a local variable

Reply:
You reviewed the NO_REVIEW commit belonging to https://github.com/linagora/james-project/pull/2524, where I had that comment already and fixed it